script.js

```diff
@@ -223,4 +223,26 @@ form.addEventListener('submit', (postIt) => {
   if (!formValidate()) {
     postIt.preventDefault();
   }
-});
\ No newline at end of file
+});
+
+// local storage start here
+
+form.addEventListener('input', () => {
+  const jsonForm = {
+    name: document.querySelector('.form-control1').value,
+    email: document.querySelector('.form-control2').value,
+    textArea: document.querySelector('.form-control3').value
+  };
+
+  localStorage.setItem('contactWay', JSON.stringify(jsonForm));
+});
+
+const nonJsonForm = JSON.parse(localStorage.getItem('contactWay'));
+if(nonJsonForm !== null) {
+  document.querySelector('.form-control1').value = nonJsonForm.name;
+  document.querySelector('.form-control2').value = nonJsonForm.email;
+  document.querySelector('.form-control3').value = nonJsonForm.textArea;
+}
+else {
+  return {}
+}
```

Comment:
- [ ] [OPTIONAL] Since you are not in a function you don't need this else case that returns an empty object

Reply:
Thanks for the review 😄